src/components/Projects/projects.css

```
@@ -0,0 +1,88 @@
+.projects {
+  display: flex;
+  flex-flow: column wrap;
+}
+
+.projects h1 {
+  text-align: end;
+  margin-bottom: 32px;
+}
+
+.projects .project {
+  display: flex;
+  align-items: center;
+  justify-content: end;
+  margin-bottom: 33px;
+}
+
+.projects .project .info-of-project {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: end;
+  text-align: end;
+}
+
+.projects .project-1,
+.projects .project-3 {
+  display: flex;
+  align-items: center;
+  justify-content: start;
+}
+
+.projects .project-1 .info-of-project,
+.projects .project-3 .info-of-project {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: start;
+  text-align: start;
+  order: 1;
+}
+
+.projects .project-0 img,
+.projects .project-2 img {
+  margin-left: 24px;
+}
+
+.projects .project-1 img,
+.projects .project-3 img {
+  margin-right: 24px;
+  order: 0;
+}
+
+.projects .project .info-of-project p {
+  width: 592px;
+}
+
+.projects .project .info-of-project .language {
+  margin-top: 13px;
+}
+
+.projects .project .info-of-project .language span {
+  border-radius: 10px;
+  font-size: 18px;
+  font-weight: 500;
+  padding: 6px 12px;
+}
+
+.projects .project .info-of-project .language .React {
```

```suggestion
.projects .react {
```

You can shortcut your css path.
Also classes are ideally kebab-case